openedx/features/portfolio_project/__init__.py

```diff
@@ -0,0 +1,13 @@
+"""
+Portfolio project helpers and settings
+"""
+from openedx.core.djangoapps.waffle_utils import (
+    CourseWaffleFlag, WaffleFlag, WaffleFlagNamespace
+)
+
+# Namespace for portfolio project waffle flags.
+WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name='portfolio_project')
+
+
+# https://openedx.atlassian.net/browse/LEARNER-3926
```

Comment:
You can leave this if you want, but a more correct way would be to have your commit comment include "LEARNER-3926" on its own line at the end, and it will appear in JIRA and from the commit comment someone could find the ticket.